backend/reservations/auth.py

```diff
@@ -14,28 +14,28 @@ def authenticate(request):
     data = request.data
     try:
         jwt_data=data["jwt"]
-    except KeyError:
-        logger.warning("JWT information missing ip:%s", request.META['REMOTE_ADDR'])
+    except KeyError as e:
+        logger.warning("[-] JWT information missing ip:%s data: %s, e: %s", request.META['REMOTE_ADDR'], data, e)
```

Comment:
I could have swore I made this comment already but that was before `N` shots of fernet....

What is the meaning of the `[-]` prefix? Is it just to stand out?